dev/tasks/generate-crds

```diff
@@ -55,7 +55,7 @@ go run ./scripts/crd-tools reflow-descriptions --dir apis/config/crd/
 
 # excluded_resources are resources that are under development for a direct conversion
 # we don't modify the CRD just yet for those but will in the future
-excluded_resources=("computeforwardingrule")
+excluded_resources=("computeforwardingrule" "certificatemanagerdnsauthorization")
 
```

Comment:
So for a beta resource, we suggest not excluding the CRD generation using `excluded_resources`, because it gives the PR reviewing no clue on whether the outcome is correct or not (also it would be very error prone for the developers).

Instead, you can add the `cnrm.cloud.google.com/dcl2crd: "true”` in the type go tag. ([example](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/196a4b9a28b59b17936a443d5b36bb65f3c42fd9/apis/apikeys/v1alpha1/apikey_type.go#L44)), and comment out those new fields that do not exist in the TF-based resources according to the outcome from dev/task/generate-crds. Guide [here](https://docs.google.com/document/d/1Az2yOr9dGHrj-IGvYEhGxjpajGmcd65mQ4OjsAQSXyE/edit?resourcekey=0-Mf91G-_QDqq6XEVzf_CnQQ&tab=t.0#bookmark=id.ubp9htt000db)